rotkehlchen/tests/utils/ethereum.py

```diff
@@ -171,7 +174,7 @@ def wait_until_all_nodes_connected(
         timeout: int = NODE_CONNECTION_TIMEOUT,
 ):
     """Wait until all ethereum nodes are connected or until a timeout is hit"""
-    connect_at_start = [x for x in connect_at_start if x.node_info.name != evm_inquirer.etherscan_node_name]  # noqa: E501
+    connect_at_start = [x for x in connect_at_start if x.node_info.name != ETHEREUM_ETHERSCAN_NODE_NAME]  # noqa: E501
```

Comment:
i think i'm confused. you deleted every etherscan node so there's no way there's going to be a match for the `if` check.

Reply:
I did the change in two steps first removing the other etherscan endpoints and finally all the etherscan from the db. THis isn't needed